### PR TITLE
Suggestion from session words

### PR DESCRIPTION
**Pull request by shouya**
See https://github.com/minad/jinx/issues/96.

### CODE REVIEW

File: jinx.el [hunk 1]
```diff
@@ -162,6 +162,11 @@ checking."
   "List of modes included by `global-jinx-mode'."
   :type '(repeat symbol))
 
+(defcustom jinx-max-distance-for-suggestion
```
Comment (shouya) [inline jinx.el:165]: I know the name of this variable is not as satisfactory. I'm expecting to change it if you could suggest a better one.

**Comment by minad** [comment]
Thanks! It looks good. Let's wait on the FSF assignment before proceeding.

**Comment by shouya** [comment]
My apologies for the delays.

Today I finally heard back from FSF and signed the assignment. I guess we can finally merge it now?

**Comment by minad** [comment]
Great, thanks! I will try this in a while, when I have a bit more free time. Your patch looks good so far.

File: jinx.el [hunk 2]
```diff
@@ -637,34 +642,51 @@ If CHECK is non-nil, always check first."
...
-                     (format #("%d " 0 3 (face jinx-key)) idx))
-                    ((< (- idx 10) (length jinx--select-keys))
-                     (format #("0%c " 0 4 (face jinx-key)) (aref jinx--select-keys (- idx 10))))))
-        sugg)
-       (cl-incf idx)
-       (puthash sugg t ht)
-       sugg)))
-   (cl-loop for (key . fun) in jinx--save-keys nconc
-            (ensure-list (funcall fun nil key word)))))
+   (jinx--correct-suggestions-from-local-words word)
```
Comment (minad) [inline jinx.el:687]: The function is called `jinx--correct-suggestions-from-session-words`.

**Comment by minad** [comment]
Merged with minor changes in https://github.com/minad/jinx/commit/55afe148c1f6fe5f5b1adb983aeaa0cc69126161. Thanks!

**Pull request closed without merging**